lib/Controller/WopiController.php

```diff
@@ -606,7 +606,10 @@ private function getFileForWopiToken(Wopi $wopi) {
 			}
 
 			$userFolder = $this->rootFolder->getUserFolder($editor);
-			$file = $userFolder->getById($wopi->getFileid())[0];
+			$files = $userFolder->getById($wopi->getFileid());
+			if (isset($files[0]) && $files[0] instanceof File) {
+				$file = $files[0];
+			}
```

Comment:
Should it not throw a NotFoundException btw?

Reply:
Yeah, but there was this sentry error where I'm not sure how it happened: https://sentry.io/share/issue/c9b243560cea45f6bad487f3f7ee6d02/

Reply:
Yes. But in this case that would mean we return null right. If there are no files retruned in getById we should throw a NotFoundException right?

Reply:
Ah right, will fix that.

Reply:
added a fixup

Reply:
Oh thanks, totally forgot about that. Rebased it